places-database-management-1/bulk-import-of-pois.md

```diff
@@ -63,15 +63,15 @@ Running the Script
 
       * `access_code`
       
-        This is your access code obtained from the call to Adobe IMS.
+        This is your access code obtained from the call to Adobe IMS. See [Prequisites for using the Places REST API](../places-rest-apis/adobe-i-o-integration/README.md) for directions on how to obtain this access code. 
       
       * `org_id`
       
-        The Experience Cloud orgID into which the POIs are to be imported.
+        The Experience Cloud orgID into which the POIs are to be imported. See [Prequisites for using the Places REST API](../places-rest-apis/adobe-i-o-integration/README.md) for directions on how to obtain the org id.
       
       * `api_key`
       
-        This is your Places REST API key obtained from your Adobe I/O Places Integration.
+        This is your Places REST API key obtained from your Adobe I/O Places Integration. See [Prequisites for using the Places REST API](../places-rest-apis/adobe-i-o-integration/README.md) for directions on how to obtain the api key.
```

Comment:
Please reword the 2nd sentence as follows:
"For information about how to obtain the API key, see [Prequisites for using the Places REST API](../places-rest-apis/adobe-i-o-integration/README.md). "